examples/rn-todo-list/frontend/__tests__/App.test.tsx

```diff
@@ -4,7 +4,10 @@
 
 import 'react-native';
 import React from 'react';
-import App from '../App';
+import {App} from '../index';
+
+// Note: import explicitly to use the types shipped with jest.
+import {it} from '@jest/globals';
```

Comment:
I wonder if this should be included in a `types` array in a tsconfig.json instead 🤔

Reply:
I'll look into it, this was the default when creating a new project (`npx react-native@latest init`).

Reply:
No worries then 👍 It's better to keep aligned with the template.